agent/xds/listeners.go

```diff
@@ -32,6 +32,8 @@ import (
 	"github.com/hashicorp/consul/sdk/iptables"
 )
 
+const virtualIPTag = "virtual"
```

Comment:
Would this be better off in `agent/structs`?

Reply:
If we don't need to export it now, I think it's desirable to be able to keep it locally unexported. We can always move it and export it later, but if we don't need to it's one less thing that has to be changed when we need to split up the `agent/structs` package.